cloudigrade/config/settings/base.py

```diff
@@ -190,3 +200,9 @@
     'account.tasks.create_volume': {'queue': 'create_volume'},
     'account.tasks.enqueue_ready_volume': {'queue': 'enqueue_ready_volumes'},
 }
+CELERYBEAT_SCHEDULE = {
+    'scale_up_inspection_cluster_every_5_min': {
+        'task': 'account.tasks.scale_up_inspection_cluster',
+        'schedule': 60 * 5,  # seconds
```

Comment:
Regarding timing, I think we originally were talking about trying to run the cluster once every 30 mins or an hour, maybe have it be configurable? Either way 5 minutes seems awfully small with how long snapshot copy operations can take.